tests/test_drs.py

```diff
@@ -629,6 +643,10 @@ def test_bucket_name_and_key(self):
             with self.assertRaises(AssertionError):
                 drs._bucket_name_and_key("not a valid gs url")
 
+    def test_get_drs_cloud_platform(self):
+        cloud_platform = drs.get_drs_cloud_platform()
```

Comment:
The execution env is google if there is a workspace bucket env variable (which is set in the test config.py file) otherwise it is azure. I couldn't quite figure out how to set the execution env to Azure in the test, but I checked by manually removing the env variable from the config file.